images/golang-dind/build.yaml

```diff
@@ -4,3 +4,8 @@ name: golang-dind # Name of the image to be built
 images:
 - ${_REGISTRY}/${_NAME}:${_DATE_STAMP}-${_GIT_REF}
 - ${_REGISTRY}/${_NAME}:latest
```

Comment:
You should update the `images` list here to include `GO_VERSION`, e.g.:

```
# Image names to be tagged and pushed
images:
- ${_REGISTRY}/${_NAME}:${_DATE_STAMP}-${_GIT_REF}-${BAZEL_VERSION}
- ${_REGISTRY}/${_NAME}:latest-${BAZEL_VERSION}
```

from: https://github.com/jetstack/testing/blob/master/images/bazelbuild/build.yaml